manifests/base/deployment/argocd-image-updater-deployment.yaml

```diff
@@ -149,6 +149,30 @@ spec:
                 name: argocd-image-updater-secret
                 key: webhook.harbor-secret
                 optional: true
+        - name: ENABLE_WEBHOOK_RATELIMIT
+          valueFrom:
+            configMapKeyRef:
+                name: argocd-image-updater-config
+                key: webhook.enable-rate-limit
+                optional: true
+        - name: WEBHOOK_RATELIMIT_NUM_ALLOWED_REQUESTS
+          valueFrom:
+            configMapKeyRef:
+                name: argocd-image-updater-config
+                key: webhook.ratelimit-num-allowed-requests
+                optional: true
```

Comment:
the above env var name should be consistent with the one used in run.go and webook.go.

the config map key should also be consistent with the rest as much as possible.